healthtools/scrapers/base_scraper.py

```diff
@@ -82,36 +85,37 @@ def scrape_site(self):
         '''
         This functions scrapes the entire website by calling each page.
         '''
-        self.set_site_pages_no()
+        self.site_pages_no = self.set_site_pages_no()
 
+        try:
+            if not self.site_pages_no:
+                raise Exception("No pages found.")
+        except Exception as err:
+            self.print_error("- ERROR: scrape_site() \n- SOURCE: {} \n- MESSAGE: {}".format(self.site_url, err))
+            return
```

Comment:
The above seems redundant. A try-catch statement is comparable to an if-else here..